lib/ios.js

```diff
@@ -0,0 +1,127 @@
+import {cp, fs, macOsxVersion} from './utils';
+import {DoctorCheck} from './doctor';
+import log from './logger';
+import path from 'path';
+import NodeDetector from './node-detector';
+let checks = [];
+
+// Helpers
+
+async function authorizeIos () {
+  try {
+    var authorizePath = path.resolve(__dirname, "../../bin", "authorize-ios.js");
+    cp.exec("'" + process.execPath + "' '" + authorizePath + "'", { maxBuffer: 524288});
```

Comment:
we'll want to bring in the work @soniayang97 is doing with authorize ios and use that here I guess